### PR TITLE
Replaced Hero.js hardcoded Circles component to object literal mapping

### PR DESCRIPTION
**Pull request by LeeJay0126**
What title says.

But we still need to find a way to replace the icons within the return component.

I tried <${icons.icon} to see if that returns an identifier, but it didn't.

We should discuss if we want to move the hardcoded object literals to our backend or database as well.

Worst comes worst, we can always use if/else statement within Circles component to identy the icons

### CODE REVIEW

File: langaratribe/src/Pages/Home/HomeComponents/Circles.js [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+const Circles = (props) => {
 
     return (
-        <div id="course"
-            onMouseEnter={() => props.setState("course")}
-            onMouseLeave={() => props.setState("default")}
-        ></div>
+        <div className="iconFlex">
+            <div id={props.id}
+                onMouseEnter={() => props.setState(`${props.id}`)}
```
Comment (Jina-sw) [inline langaratribe/src/Pages/Home/HomeComponents/Circles.js:29]: onMouseEnter= {() => {
props.setState(`${props.icon}`);
String iconName = props.icon;
return(
<div>
<{iconName}/>
</div>
)
}
}


.

Comment (Jina-sw) [reply langaratribe/src/Pages/Home/HomeComponents/Circles.js:29]: I want to look at the full useState statement too

Comment (LuSilvaDeveloper) [reply langaratribe/src/Pages/Home/HomeComponents/Circles.js:29]: Really good job Jay. You are doing really well.

**Review by LuSilvaDeveloper — COMMENTED** [review]
It's all good.

**Pull request merged**